fastlane/lib/fastlane/actions/download_dsyms.rb

```diff
@@ -98,6 +97,16 @@ def self.download(download_url, bundle_id, train_number, build_version, output_d
         Actions.lane_context[SharedValues::DSYM_PATHS] << File.expand_path(path)
       end
 
+      def self.get_version(latest_version)
+        candidate_build = latest_version.candidate_builds.first
```

Comment:
We can't use Safe navigation operator because of TargetRubyVersion 😢 
So this method is needed.